### DISCUSSION TITLE
Capacity Charge Needed

### DISCUSSION THREAD

**Discussion by Captain0Kangaroo** [nathanmarlor/foxess_em#336] [General]
Last night was the first time I'd let EM fully manage my system. Often I intervene and add a few extra Kw.

During the off peak I run the dishwasher, washing machine and top up the hot tub (all set to run 00:30 till 04:30).

This morning I'm presented with an empty battery. Is this because the system didn't change mode at 00:30 because it thought it had enough to last until morning but because my appliances started they used the battery and not grid (until the battery ran out)?

To top it off it's Foggy as hell outside too!

If this is the case could there be a setting that adds a nominal 0.5kW or something that means the house would run on grid during the off peak period?

**Comment by nathanmarlor** [comment]
@Captain0Kangaroo can you add graphs for house load, min SoC and battery SoC please. It should set the min SoC to dawn load (i.e. 4:30 -> solar takeover + buffer) so once the appliances have drained the battery to this min SoC it'll pull from the grid.

**Reply by Captain0Kangaroo** [reply]
![image](https://github.com/nathanmarlor/foxess_em/assets/96144685/b8770fb1-d5e0-4341-ae70-0ff5283e946a)

![image](https://github.com/nathanmarlor/foxess_em/assets/96144685/5c30225a-7853-48db-9066-6aae7d982fbf)

![image](https://github.com/nathanmarlor/foxess_em/assets/96144685/d383ea2e-9055-4dbc-ace0-9643f303f962)

**Comment by nathanmarlor** [comment]
@Captain0Kangaroo sorry, should've been Min SoC (On Grid).

The main issue here is that you've got the infamous battery drain issue, you can see from ~4:30 -> ~5:30 your SoC drops almost 10% without any associated house load. This can be resolved by updating the BMS to the latest firmware.

EM is doing exactly what it should, at 00:30 it appears to set the min SoC to ~22% which then allows the house to pull from the battery until this is hit then it holds. Between 01:00 -> 04:00 your SoC begins to drop off due to the draining issue, once it gets 2-3% below the min SoC EM kicks in and initiates another force charge to top it back up (~04:10). The solar begins to charge the battery at ~08:10, and if it wasn't for the phantom drop you'd still have ~20% SoC left, which assuming a 10kwh battery and a 1kwh buffer is pretty much right.

**Reply by Captain0Kangaroo** [reply]
Ok good to know.... Thank you.

Any tips on how to update the BMS? I've got an installer account - just can't remember where to do it.

**Reply by nathanmarlor** [reply]
On the Fox Cloud site there's a menu on the left for firmware updates, you want to the battery section.

**Reply by Captain0Kangaroo** [reply]
System has been updated by fox (last week).

I've tried again to just let the system deal with everything but again this morning I've got a dead battery and running from the grid.

I love the idea of this but for me it's just not working.

Is there anything I can check or try?
